envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/EnvoyDefaultFilters.kt

```diff
@@ -30,12 +33,17 @@ class EnvoyDefaultFilters(
     private val headerToMetadataHttpFilter = headerToMetadataHttpFilter(defaultHeaderToMetadataConfig)
     private val defaultHeaderToMetadataFilter = { _: Group, _: GlobalSnapshot -> headerToMetadataHttpFilter }
     private val envoyRouterHttpFilter = envoyRouterHttpFilter()
-
     /**
      * Default filters should not be private, user should have an option to pick any filter.
      * Remember: order matters.
      */
     val defaultEnvoyRouterHttpFilter = { _: Group, _: GlobalSnapshot -> envoyRouterHttpFilter }
+    val defaultRateLimitLuaFilter = { group: Group, _: GlobalSnapshot ->
```

Comment:
both can be private

Reply:
Nope. See the comment above the declarations.

Reply:
cool i am blind apparently :D